modules/allpass.h

```diff
@@ -0,0 +1,68 @@
+// #Allpass
+// Allpass filter module
+// ## Credits
+// *Original author(s) : Barry Vercoe, John ffitch
+// *Year : 1991
+// *Ported from soundpipe by Ben Sergentanis, May 2020
+
+#pragma once
+#ifndef DSY_ALLPASS_H
+#define DSY_ALLPASS_H
+
+#include <stdint.h>
+#ifdef __cplusplus
+
+#include <math.h>
+
+namespace daisysp
+{
+
+class Allpass
+{
+    public:
+    Allpass() {}
+    ~Allpass() {}
+  
+    // ### Init
+    //
+    // Initializes the allpass module.
+    //
+    // sample_rate - The sample rate of the audio engine being run. 
+    // 
+    // ~~~~
+        void Init(float sample_rate);
+    // ~~~~
+
+    // ### Process
+    //
+    // 
+    // ~~~~
+        float Process(float in);
+    // ~~~~
+
+    // #Setters
+    //
+    // ### SetLoopTime
+    //
+    // 
+    // ~~~~
+        inline void SetLoopTime(float looptime) { loop_time_ = fminf (0.5f, looptime); buf_size_ = (int) (loop_time_ * sample_rate_); }
```

Comment:
This is a bit long to be a single line. And probably complex enough just to move it into the cpp file.